### PR TITLE
feat: add clusters in the returned /api/kafkas_mgmt/v1 collection response

### PR DESCRIPTION
**Pull request by machi1990**
## Description

add clusters in the returned /api/kafkas_mgmt/v1 collection response

## Verification Steps

1. `make run`
2. `curl -XGET -http://localhost:8000/api/kafkas_mgmt/v1/ | jq` and ensures that the clusters are returned as part of the collection

## Checklist (Definition of Done)

- ~~[ ] All acceptance criteria specified in JIRA have been completed~~
- ~~[ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)~~
- ~~[ ] Documentation added for the feature~~
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- ~~[ ] Required metrics/dashboards/alerts have been added (or PR created).~~
- ~~[ ] Required Standard Operating Procedure (SOP) is added.~~
- ~~[ ] JIRA has been created for changes required on the client side~~

### CODE REVIEW

**Review by pawelpaszki — APPROVED** [review]

File: internal/kafka/internal/routes/route_loader.go [hunk 1]
```diff
@@ -269,24 +277,22 @@ func (s *options) buildApiBaseRouter(mainRouter *mux.Router, basePath string) er
...
-		Name(logger.NewLogEvent("deregister-enterprise-cluster", "deregister enterprise data plane cluster by ID").ToString()).
-		Methods(http.MethodDelete)
-	clusterRouter.HandleFunc("/{id}", clusterHandler.Get).
-		Name(logger.NewLogEvent("get-enterprise-cluster", "get an enterprise data plane cluster by ID").ToString()).
-		Methods(http.MethodGet)
-	clusterRouter.HandleFunc("/{id}/addon_parameters", clusterHandler.GetEnterpriseClusterWithAddonParams).
-		Name(logger.NewLogEvent("get-enterprise-cluster-addon-parameters", "get addon parameters of an enterprise data plane cluster by ID").ToString()).
-		Methods(http.MethodGet)
+	// /api/kafkas_mgmt/v1
+	v1Metadata := api.VersionMetadata{
```
[the remark this answers is not in the document]
Comment (machi1990) [reply internal/kafka/internal/routes/route_loader.go:281]: What do you mean by shorter URLs?

[the remark this answers is not in the document]
Comment (miguelsorianod) [reply internal/kafka/internal/routes/route_loader.go:281]: `/api/kafkas_mgmt_v1` is a shorter URL than `/api/kafkas_mgmt_v1/kafkas` for example

[the remark this answers is not in the document]
Comment (machi1990) [reply internal/kafka/internal/routes/route_loader.go:281]: Thanks. I am not sure I understand the full context. I'll reach out to get more context.

[the remark this answers is not in the document]
Comment (miguelsorianod) [reply internal/kafka/internal/routes/route_loader.go:281]: After discussing through other channels I am fine leaving it like this. The motivation on moving it below is that the collections are gradually  into the v1Collections slice over the method body and at the end the v1Metadata is built using then followed by all of the other elements that use v1Metadata

[the remark this answers is not in the document]
Comment (machi1990) [reply internal/kafka/internal/routes/route_loader.go:281]: Thanks for the feedback @miguelsorianod

**Pull request merged**